modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/message/SnapshotRequestMessage.java

```diff
@@ -19,11 +19,13 @@
 
 import java.util.UUID;
 import org.apache.ignite.network.NetworkMessage;
+import org.jetbrains.annotations.Nullable;
 
 /**
  * Base interface for snapshot request messages.
  */
 public interface SnapshotRequestMessage extends NetworkMessage {
     /** Snapshot id. */
+    @Nullable
```

Comment:
Why is it nullable? It doesn't look right because a snapshot request must reference a snapshot by ID. If something fails due to it, maybe a ticket has to be filed and linked here with a TODO to resolve the issue.